File: KatlaSport.Services.Models/CompanyStructureManagement/Department.cs
﻿
using System;

namespace KatlaSport.Services.CompanyStructureManagement
{
    public class Department
    {
        public int Id { get; set; }

        public int? ParentId { get; set; }

        public string Code { get; set; }

        public string Name { get; set; }

        public bool IsDeleted { get; set; }

        public DateTime LastUpdated { get; set; }
    }
}

File: KatlaSportNg/src/app/department-managment/models/department.ts
import { DepartmentMainInfo } from '../models/department-main-info';

export class Department {
    constructor(
        public id: number,
        public parentId: number,
        public parentName: string,
        public name: string,
        public code: string,
        public isDeleted: boolean,
        public lastUpdated: string,
        public childDepartments:DepartmentMainInfo[]
    ) { }
}
